time:
  {
    min_length: 5,
    max_length: 10
  }
rate: 200